Description: Drop sphinx_accessors as not currently present in Debian
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2021-05-10
Forwarded: not-needed

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -19,7 +19,7 @@
 import sys
 from contextlib import suppress
 
-import sphinx_autosummary_accessors
+# import sphinx_autosummary_accessors
 
 import xarray
 
@@ -75,7 +75,7 @@
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
-    "sphinx.ext.autosummary",
+#    "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.extlinks",
     "sphinx.ext.mathjax",
@@ -83,7 +83,7 @@
     "IPython.sphinxext.ipython_directive",
     "IPython.sphinxext.ipython_console_highlighting",
     "nbsphinx",
-    "sphinx_autosummary_accessors",
+#    "sphinx_autosummary_accessors",
     "sphinx.ext.linkcode",
     "sphinx_panels",
     "sphinxext.opengraph",
@@ -178,7 +178,7 @@
 mathjax_path = 'file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates", sphinx_autosummary_accessors.templates_path]
+templates_path = ["_templates" ] # , sphinx_autosummary_accessors.templates_path]
 
 # The suffix of source filenames.
 # source_suffix = ".rst"
